Writerside/topics/implementation/gtfs-schedule.md

```diff
@@ -1,3 +1,76 @@
 # GTFS Schedule
 
-Start typing here...
\ No newline at end of file
+In the public transit service, only GTFS schedule data is utilized, which provides static transit service information
+for a specified validity period, without incorporating GTFS realtime data. The GTFS reader class processes all required
+files and fields, as well as some conditionally required fields (such as transfers, bike, and accessibility
```

Comment:
maybe differentiate between optional fields (accessibility) and optional files (transfers)